test/test_datasets_utils.py

```diff
@@ -1,53 +0,0 @@
-# Copyright (C) 2021, Pyronear contributors.
```

Comment:
we still have a utils submodule if we merge this PR, so some unittests would be good :)